tasks/lib/jscs.js

```diff
@@ -71,13 +71,26 @@ exports.init = function( grunt ) {
     };
 
     /**
+     * @see Checker#fixPath
+     */
+    JSCS.prototype.fix = function( path ) {
```

Comment:
Added a unit test that makes sure it calls the checker. What else would you test?